lago/export.py

```diff
@@ -16,11 +16,12 @@
 
 class DiskExportManager(object):
     """
-    ExportManager object is responsible on the export process of an image
-    from the current Lago prefix.
+    :class:DiskExportManager object is responsible on the export process of
```

Comment:
Not sure if you need the :class: here, it is meant for cross-references(your already inside the class), also if it is needed you need '`' for sphinx to render it.